lib/aruba/api/filesystem.rb

```diff
@@ -372,10 +372,25 @@ def with_file_content(file, &block)
       #
       # @result [FileSize]
       #   Bytes on disk
+
+      TYPICAL_FS_UNIT = 4096 # Very typical, except for giant or embedded filesystems
```

Comment:
Are those constants used somewhere else? If not, let's make them just variables. I find that easier to maintain.